docs/index.html

```diff
@@ -623,10 +623,10 @@ <h3 id="building-polaris">Building Polaris</h3>
 &lt;/code&gt;&lt;/pre&gt;
 &lt;p&gt;This will create a new catalog called &lt;strong&gt;quickstart_catalog&lt;/strong&gt;.&lt;/p&gt;
 &lt;p&gt;The &lt;code&gt;DEFAULT_BASE_LOCATION&lt;/code&gt; you provide will be the default location that objects in this catalog should be stored in, and the &lt;code&gt;ROLE_ARN&lt;/code&gt; you provide should be a &lt;a href=&quot;https://docs.aws.amazon.com/IAM/latest/UserGuide/reference-arns.html&quot;&gt;Role ARN&lt;/a&gt; with access to read and write data in that location. These credentials will be provided to engines reading data from the catalog once they have authenticated with Polaris using credentials that have access to those resources.&lt;/p&gt;
-&lt;p&gt;If you’re using a storage type other than S3, such as Azure, you’ll provide a different type of credential than a Role ARN. For more details on supported storage types, see the &lt;a href=&quot;./entities.md#storage-type&quot;&gt;docs&lt;/a&gt;.&lt;/p&gt;
-&lt;p&gt;Additionally, if Polaris is running somewhere other than &lt;code&gt;localhost:8181&lt;/code&gt;, you can specify the correct hostname and port by providing &lt;code&gt;--host&lt;/code&gt; and &lt;code&gt;--port&lt;/code&gt; flags. For the full set of options supported by the CLI, please refer to the &lt;a href=&quot;./command-line-interface.md&quot;&gt;docs&lt;/a&gt;.&lt;/p&gt;
+&lt;p&gt;If you’re using a storage type other than S3, such as Azure, you’ll provide a different type of credential than a Role ARN. For more details on supported storage types, see the &lt;a href=&quot;#storage-type&quot;&gt;docs&lt;/a&gt;.&lt;/p&gt;
+&lt;p&gt;Additionally, if Polaris is running somewhere other than &lt;code&gt;localhost:8181&lt;/code&gt;, you can specify the correct hostname and port by providing &lt;code&gt;--host&lt;/code&gt; and &lt;code&gt;--port&lt;/code&gt; flags. For the full set of options supported by the CLI, please refer to the &lt;a href=&quot;https://github.com/apache/polaris/blob/main/docs/command-line-interface.md&quot;&gt;docs&lt;/a&gt;.&lt;/p&gt;
```

Comment:
If I am reading this correctly, clicking a link in a doc on the website will lead you to a doc on github? That feels bad

Reply:
To add to it, it points to the main branch only, which breaks other branches.
```
https://github.com/apache/polaris/blob/main/docs/...
```

Reply:
I'd recommend to move forward with a way to support a relative paths. We can play with building tools to support it.